app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +92,34 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for collect_all_items is too high. [28.91/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [24/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)